movies-library/src/components/Counter.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+export class Counter extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      count: this.props.initialCount,
+    };
+  }
+
+  increment() {
+    this.setState({
+      count: this.state.count + 1,
+    });
+  }
+
+  decrement() {
+    this.setState({
+      count: this.state.count - 1,
+    });
+  }
+
+  render() {
+    return React.createElement(
+      "div",
+      { style: { color: "red", marginLeft: "60px" } },
+      <div>
+        <p>Current Count: {this.state.count}</p>
+        <button className="inc" onClick={(e) => this.increment()}>
```

Comment:
This is not a good practice to create custom style for each button they will be same through all project. Better to create Button component and put those inside. Later on for different types of button we could implement type prop. In general - if something can be moved into separate component - it should be.

Reply:
I created reusable button component